### DISCUSSION TITLE
Disable BracketHighlighter using a .sublime-project file

### DISCUSSION THREAD

**Discussion by fnando** [facelessuser/BracketHighlighter#626] [Q&A]
Hi,

Is it possible to disable BH by using a .sublime-project file? 

Unfortunately, `ignore_packages` doesn't work per project, so it'd require a configuration that BH can read. The way GitGutter does it, for instance, is by also reading the prefixed global `git_gutter_*` config.

**Comment by facelessuser** [comment]
No, not currently. I've never seen a request for such until now. I think this is not a community desired feature, at least for BracketHighlighter.
